satpy/readers/mirs.py

```diff
@@ -18,16 +18,20 @@
 """Interface to MiRS product."""
 
 import os
+import re
 import logging
 import datetime
+import appdirs
 import numpy as np
 import xarray as xr
 import dask.array as da
 from satpy import CHUNK_SIZE
 from satpy.readers.file_handlers import BaseFileHandler
+from satpy.aux_download import find_registerable_files, retrieve
 
 LOG = logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG)
+SATPY_DATA_DIR = appdirs.user_data_dir("satpy")
```

Comment:
I don't think you're using this anymore or appdirs (imported above). If you use `retrieve` you shouldn't need the data directory anyway. `retrieve` should give you the full path.